TestProviders/ReadyProvider.psm1

```diff
@@ -36,9 +36,31 @@ Class ReadyProvider : TestProvider
 			Add-Member -InputObject $objNode -MemberType NoteProperty -Name UserName -Value $null -Force
 			Add-Member -InputObject $objNode -MemberType NoteProperty -Name Password -Value $null -Force
 			Add-Member -InputObject $objNode -MemberType NoteProperty -Name RoleName -Value $null -Force
+			Add-Member -InputObject $objNode -MemberType NoteProperty -Name InternalIP -Value $null -Force
 			return $objNode
 		}
 
+		function GetIPAddressFromIpAddrInfo([string] $ipAddrInfo) {
+			Write-LogDbg "GetIPAddress $ipAddrInfo"
+			[regex] $re = "(?:[0-9]{1,3}\.){3}[0-9]{1,3}"
+			[string] $matchedIp = $re.Match($ipAddrInfo)
+			return $matchedIp
+		}
+
+		function SetInternalIPAddress([object] $AllVMData) {
+			$count = 0
+			foreach ($vmData in $AllVMData) {
+				$ipAddrInfo = Run-LinuxCmd -username $global:user -password $global:password -ip $($vmData.PublicIp) -port $($vmData.SSHPort) -command "ip -4 address | awk -F': ' '!/lo/ {print `$2}' | xargs ip address show" -RunAsSudo
+				$ipAddress = GetIPAddressFromIpAddrInfo -ipAddrInfo $ipAddrInfo
+				if ($ipAddress) {
+					$AllVmData[$count].InternalIP = $ipAddress
+				} else {
+					Write-LogErr "Cannot get the internal IP address for $($vmData.PublicIp):$($vmData.SSHPort)"
```

Comment:
Change "internal IP address" to "Private IP address" in the consistent manner of LISAv2 logging.

Reply:
LISAv2 mised Private and Internal. No need to change. Don't see consistency any more.